modules/server/src/app.js

```diff
@@ -10,7 +10,10 @@ app.use(cors());
 export default async function (rootPath = '') {
 	global.__basedir = rootPath;
 
-	return Arranger({ enableAdmin: ENV_CONFIG.ENABLE_ADMIN }).then((router) => {
+	return Arranger({
+		enableAdmin: ENV_CONFIG.ENABLE_ADMIN,
+		enableAggregationMode: ENV_CONFIG.ENABLE_AGGREGATION_MODE,
```

Comment:
tempted to suggest/request a code comment on what "aggregation" means in this context, as it could be mistaken as a toggle to enable the existing facets' aggregations.

Reply:
would renaming the env var to ENABLE_AGGREGATION_ONLY_MODE improve it at all?

Reply:
As far as adding comments, these should be JSDoc style comments on the Arranger function. This is a good place to describe the purpose of all properties of the configuration object.